src/components/Tooltip/Tooltip.tsx

```diff
@@ -58,12 +58,16 @@ export const Tooltip = chakra(
 
     return (
       <ChakraTooltip
-        hasArrow
         aria-label={typeof content !== "string" ? "Tooltip" : undefined}
-        label={content}
+        closeDelay={750}
```

Comment:
Does the addition of these new props act as a backup to ensure that the tooltip does close eventually?  Meaning, does the bug still exist and we are just adding something to gracefully get around the bug?  Or does this resolve the bug altogether?

Reply:
Yep, pretty much works as a backup to ensure it closes. That being said, it does resolve the bug but in a roundabout way.